config/routes.rb

```diff
@@ -1,5 +1,16 @@
 Rails.application.routes.draw do
   root to: "books#index"
 
+  resources :passwords, only: [:create, :new]
+  resource :session, only: [:create]
+
+  resources :users, only: [:create] do
+    resource :password, only: [:create, :edit, :update]
+  end
+
+  get "/sign_in" => "sessions#new", as: "sign_in"
+  delete "/sign_out" => "sessions#destroy", as: "sign_out"
+  get "/sign_up" => "users#new", as: "sign_up"
+
   resources :books, only: [:index, :new, :edit, :create, :update, :destroy]
```

Comment:
I think you should extract an `admin` namespace for the routes that are admin specific. This will make it easier to find them and to see which actions can only be performed by an admin